### DISCUSSION TITLE
How to configure clipboard for multiplexing?

### DISCUSSION THREAD

**Discussion by tommyalatalo** [Slackadays/Clipboard#183] [Q&A]
I use tmux, and as such I move between multiple terminal windows constantly, how do I configure clipboard so that the data in my clipboards is globally available across all terminal windows? I noticed that this doesn't happen by default as doing `cb show` in two windows gives me different results.

**Comment by Slackadays** [comment]
That's weird, because the clipboards should all be the same everywhere. Are you using containers at all? If so, then the clipboard will only go in that container. Try `cb info` and see if the storage location is different in your windows
